src/sentry/integrations/github/webhook.py

```diff
@@ -298,28 +328,47 @@ def _handle(self, integration, event, organization, repo, host=None):
 
 
 class PullRequestEventWebhook(Webhook):
-    # https://developer.github.com/v3/activity/events/types/#pullrequestevent
-    def is_anonymous_email(self, email):
+    """https://developer.github.com/v3/activity/events/types/#pullrequestevent"""
+
+    def is_anonymous_email(self, email: str) -> bool:
         return email[-25:] == "@users.noreply.github.com"
 
-    def get_external_id(self, username):
+    def get_external_id(self, username: str) -> str:
         return f"github:{username}"
 
-    def get_idp_external_id(self, integration, host=None):
-        return options.get("github-app.id")
-
-    def _handle(self, integration, event, organization, repo, host=None):
+    def get_idp_external_id(self, integration: Integration, host: str | None = None) -> str:
+        # Explicitly typing to satisfy mypy.
+        external_id: str = options.get("github-app.id")
+        return external_id
+
+    def _handle(
+        self,
+        integration: Integration,
+        event: Mapping[str, Any],
+        organization: Organization,
+        repo: Repository,
+        host: str | None = None,
+    ) -> None:
         pull_request = event["pull_request"]
         number = pull_request["number"]
         title = pull_request["title"]
         body = pull_request["body"]
         user = pull_request["user"]
 
-        # The value of the merge_commit_sha attribute changes depending on the state of the pull request. Before a pull request is merged, the merge_commit_sha attribute holds the SHA of the test merge commit. After a pull request is merged, the attribute changes depending on how the pull request was merged:
-        # - If the pull request was merged as a merge commit, the attribute represents the SHA of the merge commit.
-        # - If the pull request was merged via a squash, the attribute represents the SHA of the squashed commit on the base branch.
-        # - If the pull request was rebased, the attribute represents the commit that the base branch was updated to.
-        # https://developer.github.com/v3/pulls/#get-a-single-pull-request
+        """
+        The value of the merge_commit_sha attribute changes depending on the
+        state of the pull request. Before a pull request is merged, the
+        merge_commit_sha attribute holds the SHA of the test merge commit.
+        After a pull request is merged, the attribute changes depending on how
+        the pull request was merged:
+        - If the pull request was merged as a merge commit, the attribute
+         represents the SHA of the merge commit.
+        - If the pull request was merged via a squash, the attribute
+         represents the SHA of the squashed commit on the base branch.
+        - If the pull request was rebased, the attribute represents the commit
+         that the base branch was updated to.
+        https://developer.github.com/v3/pulls/#get-a-single-pull-request
+        """
```

Comment:
I don't think this code comment is of any use to a caller of `_handle`, so I don't think it's an appropriate docstring.

Reply:
I agree but when I was working on the revert I realized this isn't being detected as docstring it's just a multi-line string which I think is acceptable comment syntax.

Reply:
I've not seen people use multi-line strings like that but I don't have a strong opinion on it.